deps/jansson/CMakeLists.txt

```diff
@@ -701,5 +701,11 @@ set(JANSSON_LIBRARIES
     jansson
     CACHE STRING "Jansson libraries")
 
+if (NOT MSVC)
```

Comment:
@RytoEX aren't we using pre-compiled Jansson on Windows already?

Reply:
Yes, but I keep the change except if jansson get removed form the repo.

Reply:
Also maybe PPA build use it, I didn't check.